README.md

```diff
@@ -1,2 +1,117 @@
 # javascript-capstone
-JavaScript Capstone Project to Build an API-based Web App
+
+> JavaScript Capstone Project to Build an API-based Web App
+[Collaborative Project]
+
+<img src="./src/assets/images/homepage-screenshot.png" width=350 height="250">
+<img src="./src/assets/images/popup-screenshot.png" width=350 height="250">
+
+
+This is a collaborative capstone proeject where the team is tasked to build a web app that displays list of food items form The Meals DB API and create and display Usedr reactions such Like and Comment using Microverse Involvement API.
+
+In this project, we have:
+
+- Used JavaScript to make websites dynamic and build basic single  page apps.
+- Used ES6 syntax.
+- Used ES6 modules.
+- Used callbacks and promises.
+- Used webpack.
+- Applied JavaScript best practices and language style guides in code.
+- Used AAA pattern for unit tests.
+- Wrote units tests for a JavaScript app.
+- Follow Gitflow.
+- Solved simple git conflicts.
+- Sent and receive data from an API.
+- Used API documentation.
+- Understood and Used JSON.
+- Made JavaScript code synchronous.
+- Performed a code review for a team member.
+
+## Live Demo
+Click [here](https://francisuloko.github.io/javascript-capstone/dist/) to visit the app
```

Comment:
- [ ] The comment functionality doesn't work on the live version. Kindly check it.